script.js

```diff
@@ -0,0 +1,35 @@
+const key = "36150daea36692d65248e6c1d9a32902";
+let weatherInfo = document.getElementById("weather-info");
+let searchBtn = document.getElementById("search-btn");
+let city = document.getElementById("city");
+
+let getWeather = () => {
+  let cityValue = city.value;
+  if (cityValue.length == 0) {
+    weatherInfo.innerHTML = `<h3>Please enter a city name</h3>`;
```

Comment:
An error message is technically not a heading. It words but would be better to use a <span> with text inside of it and add a class to it that you style in CSS